Header header
int16 hip_angle
int16 desired_hip_angle
uint16 time
int16 knee_angle
int16 desired_knee_angle
int16 PWM10000_knee
int16 PWM10000_hip
int32 velocity_knee1000
int32 velocity_hip1000



